opengrok-indexer/src/main/java/org/opengrok/indexer/analysis/Ctags.java

```diff
@@ -413,7 +413,7 @@ private void addTerraformSupport(List<String> command) {
          * the possibility of leaving out some matches.
          */
         command.add("--regex-terraform=" +
-                "/[[:<:]]resource[[:space:]]*\"([[:alpha:]][-_[:alpha:]]*)\"[[:space:]]*" +
+                "/[[:space:]]*resource[[:space:]]*\"([[:alpha:]][-_[:alpha:]]*)\"[[:space:]]*" +
```

Comment:
It seems that ctags on windows have some problem with this regex, maybe it doesn't support POSIX? However, there is no warning, only the line is not matched. I'll investigate tomorrow.